nyalib/bot_config.py

```diff
@@ -59,18 +60,23 @@ def cogs(self):
 
             if os.getenv("DEBUG"):
                 cogs.append({"file": "jishaku", "in_cogs": False})
-        else:
-            if 'owner' not in cogs:
-                cogs.append('owner')
-            if os.getenv("DEBUG") and "jishaku" not in cogs:
-                cogs.append("jishaku")
-            cogs = [
-                {"file": f.replace('.py', ''), "in_cogs": os.path.isfile(os.path.join("cogs", f))}
-                for f in cogs
-                if f.split(".")[-1] == "py" and f.split(".")[-2] != "base_cog"
-            ]
 
-        return cogs
+            return cogs
+
+        if 'owner' not in cogs and os.path.isfile(os.path.join("cogs", "owner")):
+            # Append owner cog
+            cogs.append('owner')
+
+        if os.getenv("DEBUG") and "jishaku" not in cogs:
```

Comment:
what is 'jishaku' and what are you doing with this here?

Reply:
it's a helper cog which allows code evaluation and other sorts of testing